addons/website_blog/controllers/website.py

```diff
@@ -0,0 +1,14 @@
+from odoo.addons.website.controllers import main
+
+
+class Website(main.Website):
+    def _get_robots_directives(self):
+        config = super()._get_robots_directives()
+
+        disallow_patterns = [
+            '/blog/*/tag/',
```

Comment:
`/blog/*/tag/*,*`

We want to support one tag indexing no? they are nofollow in this case on the page at least.